opl/generators/inventory_ingress.py

```diff
@@ -79,9 +80,9 @@ def _data(self):
             'system_purpose': json.dumps(self._get_system_purpose()),
             'ansible': json.dumps(self._get_ansible()),
             'insights_id': self._get_uuid(),
-            'ipv4_addr': self._get_ipv4(),
-            'ipv6_addr': self._get_ipv6(),
-            'mac_addr': self._get_mac(),
+            'ipv4_addr': self._get_ipv4(self.ip_v4_addresses),
```

Comment:
Something like this:
```suggestion
            'ipv4_addr': [self._get_ipv4() for _ in range(self.ip_v4_addresses)],
```